submissions/iakow/a-tiny-js-work/index.js

```diff
@@ -0,0 +1,63 @@
+const man = {
+   species: 'human',
+   name: 'Donald',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Aloha!',
+};
+
+const woman = {
+   species: 'human',
+   name: 'Eva',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Hello',
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Murka',
+   gender: 'female',
+   legs: 4,
+   hands: 0,
+   saying: 'Meow...',
+};
+
+const dog = {
+   species: 'dog',
+   name: 'Gektor',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'WOFFF!',
+};
+
+const catWoman = {
+   species: 'humacat',
+   name: 'Akrum',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: cat.saying,
+};
+
+man.friends = [woman, dog];
+woman.friends = [man, cat];
+dog.friends = [man, woman, catWoman, cat];
+catWoman.friends = [cat, woman];
+
+const printInhabitant = obj => {
+   const props = Object.keys(obj).map(key => {
+      if (key === 'friends') {
+         return obj[key].map(friend => friend.name).join(', ');
+      }
+
+      return obj[key];
+   })
+
+   print(props.join('; '));
+}
```

Comment:
`printInhabitant` does two major things: (1) constructs an inhabitant's presentation string AND (2) prints it.
Following [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single-responsibility_principle) makes code scalable, maintainable and less error prone.
Please refactor this.

Reply:
I think, done.